src/components/layout/mainNavbar.module.css

```diff
@@ -0,0 +1,56 @@
+.header {
+  width: 100%;
+  background-color: rgb(235, 235, 235);
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.2);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  font-family: "Lato", sans-serif;
```

Comment:
from where did you copy this code? do you understand it?

Reply:
Sure, I did copy that from the logo w3school, but I will fix it.